README.md

```diff
@@ -39,17 +39,16 @@ OpenAI API Offical Documentation https://platform.openai.com/docs/api-reference/
 
 ```scala mdoc:compile-only 
 import sttp.client4._
-import sttp.openai.OpenAI
+import sttp.openai.OpenAISyncClient
 import sttp.openai.requests.completions.chat.ChatRequestResponseData.ChatResponse
 import sttp.openai.requests.completions.chat.ChatRequestBody.ChatBody
 import sttp.openai.requests.completions.chat.Message
 
-// Create an instance of OpenAI providing your API secret-key
-
-val openAi: OpenAI = new OpenAI("your-secret-key")
+// Create an instance of OpenAISyncClient providing your API secret-key and sync backend instance
+val backend: SyncBackend = DefaultSyncBackend()
```

Comment:
this step can be ommited, by creating the client using the default-backend factory method